bundles/framework/myplaces3/view/MainView.js

```diff
@@ -162,25 +164,32 @@ Oskari.clazz.define('Oskari.mapframework.bundle.myplaces3.view.MainView',
          * @param {Oskari.mapframework.bundle.myplaces3.model.MyPlace} place prepoluate form with place data (optional)
          */
         showPlaceForm: function (location, place) {
-            const me = this;
             let layerId;
+            const me = this;
             const sandbox = this.instance.sandbox;
+            const categoryHandler = this.instance.getCategoryHandler();
+            this.categories = categoryHandler.getAllCategories();
```

Comment:
If there's not a good reason to use `this` just do a `const` instead.

Reply:
Changed to `const`.